test/helpers/nock_helper.js

```diff
@@ -28,6 +28,45 @@ class NockHelper {
     }
   }
 
+  mockOauthEndpoint(path, data) {
+    return () => {
+      nock.disableNetConnect()
+      nock('http://api.hubapi.com')
+        .get(path)
+        .reply(200, data)
+    }
+  }
+
+  mockPostOauthEndpoint(path, data, query = {}) {
+    return () => {
+      nock.disableNetConnect()
+      nock('http://api.hubapi.com', { encodedQueryParams: true })
+        .post(path, data)
+        .query(query)
+        .reply(200, data)
+    }
+  }
+
+  mockPutOauthEndpoint(path, data, query = {}) {
+    return () => {
+      nock.disableNetConnect()
+      nock('http://api.hubapi.com', { encodedQueryParams: true })
+        .put(path, data)
+        .query(query)
+        .reply(200, data)
+    }
+  }
+
+  mockFuzzyPutOauthEndpoint(path, regex, data, query = {}) {
```

Comment:
I don't love having all these helper methods and this may be due a refactor.

Reply:
Delaying the refactoring until you know a bit more about what's truly helpful in the tests sounds reasonable to me 👍